index.html

```diff
@@ -0,0 +1,148 @@
+          <li>
+            <button class="icon-btn">
+              <a class="nav-link" href="./pages/cart.html"
+                ><ion-icon class="btn-icon" name="cart-outline"></ion-icon
+              ></a>
+              <span class="notif-badge">5</span>
+            </button>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <div class="main-wrapper">
+      <section class="hero mg-bottom-s">
+        <div class="hero-section">
+          <img class="hero-img" src="assets/images/main-product.jpeg" />
+        </div>
+        <div class="hero-content">
+          <div class="hero-heading centered-text heading-l">
+            Discover the latest in fashion
+          </div>
+          <button class="hero-btn btn btn-medium btn-primary">
+            <a href="pages/products.html">Explore More</a>
+          </button>
+        </div>
+      </section>
+      <div class="heading-m mg-bottom-s centered-text">Featured Categories</div>
+      <section class="categories-display mg-bottom-l">
+        <div class="card-vertical grow feature-card">
+          <div class="card-image-container">
+            <img
+              class="card-image img-responsive"
+              src="../assets/images/product-9.jpeg"
+              alt="woman in red dress"
+            />
+          </div>
+          <div class="card-content">
+            <h2 class="heading-xxs">Spring Collection</h2>
+            <p class="para-s">From <strong>$14.99</strong> onwards</p>
+            <div class="para-xs">Fast delivery</div>
+          </div>
+        </div>
+
+        <div class="card-vertical grow feature-card">
+          <div class="card-image-container">
+            <img
+              class="card-image img-responsive"
+              src="../assets/images/product-1.jpeg"
+              alt="woman in red dress"
+            />
+          </div>
+          <div class="card-content">
+            <h2 class="heading-xxs">Prom Collection</h2>
+            <p class="para-s">From <strong>$24.99</strong> onwards</p>
+            <div class="para-xs">Fast delivery</div>
+          </div>
+        </div>
+
+        <div class="card-vertical grow feature-card">
+          <div class="card-image-container">
+            <img
+              class="card-image img-responsive"
+              src="../assets/images/product-10.jpeg"
+              alt="woman in red dress"
+            />
+          </div>
+          <div class="card-content">
+            <h2 class="heading-xxs">Fall Collection</h2>
+            <p class="para-s">From <strong>$24.99</strong> onwards</p>
+            <div class="para-xs">Fast delivery</div>
+          </div>
+        </div>
+      </section>
+    </div>
+    <footer class="footer">
+      <p class="footer-text mg-bottom-xs">
+        Made with <ion-icon name="heart"></ion-icon> by Snigdha Kamal
+      </p>
+      <ul class="social-icons mg-bottom-xs">
+        <li>
+          <a class="social-link" href="https://github.com/Snigdha17"
```

Comment:
providing a target="_blank" is better for Social hyperlinks

Reply:
done